### ISSUE TITLE
implement /travel

### ISSUE THREAD

**Issue by ThomasKnoops** [IGLADI/Ethereal-Hyperspace-Battleships#43]
Make it work to go to other coordinates than where you start...

**Comment by ThomasKnoops** [comment]
With the adding of the new database instead of the data.py file, is everything broken... Making a fix

**Comment by ThomasKnoops** [comment]
This is on hold untill the database is completely in order. Otherwise 2 people will do the exact same thing. So I'm waiting on Luca to finish.
